tests/unit/test_basic_functionality.py

```diff
@@ -678,3 +747,13 @@ def test_repr_display() -> None:
     lct.update({"Key1": "value_ov_1"}, layer="override_1", source="ov1_src")
     lct.update({"Key1": "value_ov_2"}, layer="override_2", source="ov2_src")
     assert repr(lct) == textwrap.dedent(expected_repr)
+
+
+def test_get_from_layer_deprecation() -> None:
```

Comment:
this is kinda silly - happy to remove

Reply:
I like this test but it also seems unnecessary to have in the future.